unicorecmsmama/static/css/style.css

```diff
@@ -15,6 +15,7 @@
 /*  LOCALE  */
 
 	.locale { padding: 5px 16px; text-align: right; background: #f0f0f0; border-bottom: 1px solid #e6e6e6; color: #e6e6e6; }
+	[dir='rtl'] .locale { padding: 5px 16px; text-align: left; background: #f0f0f0; border-bottom: 1px solid #e6e6e6; color: #e6e6e6; }
```

Comment:
Is good, but: you only need to apply the overrides here. All the stuff in `.locale` will automatically be applied to `[dir='rtl'] .locale` (Because a thing that matches `[dir='rtl'] .locale` will also match `.locale`).

So, this could just be:

> [dir='rtl'] .locale { text-align: left;}